### ISSUE TITLE
[Feature request] sort and move macros

### ISSUE THREAD

**Issue by GhbSmwc** [notepad-plus-plus/notepad-plus-plus#8074]
this:
![image](https://user-images.githubusercontent.com/13095760/77676684-08e7a400-6f65-11ea-9945-071e565a24e6.png)
On windows, the system environment variables at least lets you move each “command”:
![image](https://user-images.githubusercontent.com/13095760/77676825-3fbdba00-6f65-11ea-81ab-0dd7f17402eb.png)

And also the ability to sort by description alphabecially in case the user have lots of macros.

**Comment by sasumner** [comment]
While it certainly would be nice to have a great user interface for arranging the ordering of macros, it's worth noting that you can currently change the ordering of macros by editing the text of shortcuts.xml and physically moving the text for them around.

**Comment by GhbSmwc** [comment]
Sorry for the late reply, I did check, and did not see the list of macros I created:
![image](https://user-images.githubusercontent.com/13095760/79386394-0f4fb700-7f38-11ea-9014-abcb1cbe916d.png)

**Comment by sasumner** [comment]
@GhbSmwc You are probably looking at the wrong `shortcuts.xml`.  Check the one under `%APPDATA%`.

**Comment by GhbSmwc** [comment]
Thanks. Kinda unintuitive, hopefully the upcomming update lets the user move them around.

**Comment by sasumner** [comment]
>Kinda unintuitive, 

Agreed that it is a deficiency.  What about arranging them with submenus as well?

>hopefully the upcomming update lets the user move them around.

No code changes have been made concerning it, so I'd say the likelihood hovers near 0%.

**Comment by szaboch** [comment]
I'd like to be assigned this issue if possible. I've already implemented/tested the move up/down buttons locally and am currently implementing the sort button

**Comment by sasumner** [comment]
@szaboch It doesn't have to be assigned to you; you are free to submit a PR if you'd like.
